### ISSUE TITLE
BibXML output: IANA references should not contain dates

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-ietf#108]
See https://github.com/ietf-tools/bibxml-service/issues/312 .

In accordance with https://www.rfc-editor.org/styleguide/part2/#ref_iana_reg , the BibXML output shall not have any date.

Current:
```xml
<reference anchor="IANA_mud" target="http://www.iana.org/assignments/mud">
  <front>
    <title>Manufacturer Usage Description (MUD)</title>
    <author>
      <organization abbrev="IANA">Internet Assigned Numbers Authority</organization>
    </author>
    <date day="27" month="June" year="2018"/>
  </front>
</reference>
```

Should be:
Current:
```xml
<reference anchor="IANA_mud" target="http://www.iana.org/assignments/mud">
  <front>
    <title>Manufacturer Usage Description (MUD)</title>
    <author>
      <organization abbrev="IANA">Internet Assigned Numbers Authority</organization>
    </author>
  </front>
</reference>
```

(There is no change to Relaton data)

**Comment by strogonoff** [comment]
Is this gem responsible for generating XML? Because if we are talking about BibXML service, this will certainly be addressed in relaton-py, in `relaton.serializers.bibxml` function.

**Comment by ronaldtse** [comment]
> Is this gem responsible for generating XML? 

Yes. Metanorma uses Relaton-IETF to generate BibXML to be used within RFC XML.

This is separate from the BibXML Service.

**Comment by andrew2net** [comment]
@ronaldtse do we need to implement this behavior in this gem or in the relaton-iana?

If we need to implement it in this gem then I have the following question:
This gem creates doc identifiers with types `RFC` (for RFC, BCP, FYI, and STD docs), `Internet-Draft` (for I-D docs), or `IETF` (for the rest of the documents). So we don't know if a type of document is IANA or not. Should this gem create doc identifiers with an actual type (IANA, 3GPP, etc)?

**Comment by ronaldtse** [comment]
1. I want relaton-data-iana to contain the dates.
2. Only when in BibXML format, the IANA references do not have dates. The relaton-ietf gem generates BibXML format. Right?

**Comment by andrew2net** [comment]
> 2\. Only when in BibXML format, the IANA references do not have dates. The relaton-ietf gem generates BibXML format. Right?

Actually No. The relaton-bib generates and parses BibXML, but flavor gems can override some methods.
So there are 3 options:
- Fix the issue in the relaton-bib. In this case, any IANA bibliographic item will produce BibXML without date.
- Fix this issue in the relaton-ietf gem. In this case only IANA items processed by the relaton-ietf will be without date.
- Fix this issue in the relaton-iana gem. In this case only IANA items processed by the relaton-iana will be without date.

**Comment by ronaldtse** [comment]
@andrew2net I think the first one is the best:

> Fix the issue in the relaton-bib. In this case, any IANA bibliographic item will produce BibXML without date.

**Comment by andrew2net** [comment]
fixed in v 1.13.9
